src/Views/GameController/GameController.tsx

```diff
@@ -60,6 +62,8 @@ export const GameController: FunctionComponent = () => {
     dispatch(
       robotSimulatorSlice.actions.setPower({ channel: 1, power: leftPower })
     );
+    vm.robot.setMotorPower(0, rightPower);
+    vm.robot.setMotorPower(1, leftPower);
```

Comment:
We call this function in multiple places, and each time we also update power on `robotSimulatorSlice`. So maybe we should update `robotSimulatorSlice` from within the `vm` to make sure we always have robot power and the `robotSimulatorSlice` in sync.